lib/util/http/index.ts

```diff
@@ -44,8 +43,9 @@ function cloneResponse<T extends Buffer | string | any>(
   // Don't use json clone for buffers
   return {
     statusCode,
-    body: body instanceof Buffer ? (body.slice() as T) : clone<T>(body),
-    headers: clone(headers),
+    body:
+      body instanceof Buffer ? (body.slice() as T) : structuredClone<T>(body),
+    headers: structuredClone(headers),
```

Comment:
i think we no longer need the condition, structuredClone should handle it

Reply:
I've reverted this to use our custom `clone` for now, as there are issues with `nock` matchers.